config/routes.rb

```diff
@@ -6,4 +6,5 @@
   }
   root controller: :campaigns, action: :index
   resources :campaigns, only: [:index, :create, :new, :show]
+  resources :performers, only: [:new, :create, :update, :show]
```

Comment:
We don't really need update or show at this stage, but for future features they are needed.